scripts/Test-LowMemory.ps1

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+# This script is used as part of the systematic testing of the
+# low memory handling.
+# First it will create a list of all the tests in the test binary.
+# Second it will execute each test in the test binary.
+# Third it will check the output of each test to see if it passed or failed.
+# Fourth re-run the failed tests.
+# Fifth it will check if all tests passed and exit.
+#
+
+param ($TestProgram, $StackDepth)
+
+# Gather list of all possible tests
+$tests = & $TestProgram "--list-tests" "--verbosity=quiet"
+
+$env:EBPF_LOW_MEMORY_SIMULATION = $StackDepth
+$env:EBPF_ENABLE_WER_REPORT = "yes"
+
+Set-Content -Path ($TestProgram +".passed.log") ""
+
+# Rerun failing tests until they pass
```

Comment:
If they never pass, this will keep going until the CI/CD times out, and cancels the job, yes?